src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerListener.java

```diff
@@ -62,21 +62,20 @@ public void playerTeleport(PlayerTeleportEvent event) {
 
     @EventHandler(priority = EventPriority.LOW)
     public void playerBucketFill(PlayerBucketFillEvent event) {
-        Location translatedLocation = this.getTranslatedLocation(event.getBlockClicked(), event.getBlockFace());
         this.plugin.log(Level.FINER, "Fill: ");
         this.plugin.log(Level.FINER, "Block Clicked: " + event.getBlockClicked() + ":" + event.getBlockClicked().getType());
-        this.plugin.log(Level.FINER, "Translated Block: " + event.getPlayer().getWorld().getBlockAt(translatedLocation) + ":" + event.getPlayer().getWorld().getBlockAt(translatedLocation).getType());
+        // PlayerBucketFillEvent.getBlockClicked() returns the block that filled the bucket, so no need to translate the location!
```

Comment:
This comment is going to not make any sense without the code that is being removed. :)

Reply:
Got rid of it 👍.